tests/late_joiners.py

```diff
@@ -186,7 +186,7 @@ def run(args):
                     if i % 2 != 0 and args.consensus == "pbft":
                         continue
                     LOG.success(f"Will suspend node with id {node.node_id}")
-                    t = random.uniform(1, 10)
+                    t = random.uniform(1, 2)
```

Comment:
Why is the value 2 here? This seems like it should be calculated on the basis of the pbft timeout?

Reply:
The first timeout indicates how many seconds to wait until we first suspend the node(s). The second timeout, which is calculated based on the pbft timeout, is how many seconds we let the node(s) stay suspended before resuming, causing view changes. If we wait too long before we suspend the nodes in the first place then the test might finish without view changes actually happening.